notebooks/leb_salt_evaluation.ipynb

```diff
@@ -0,0 +1,1628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #71.            batch_size = 8
</pre><p>This could be an optional argument to <code>__init__</code>, so that we set <code>self.batch_size</code>  and it defaults to 8 (since we might run this pipeline on different machine configurations.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/leb/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>